pkg/controller/kieapp/constants/constants.go

```diff
@@ -51,8 +51,8 @@ var VersionConstants = map[string]*api.VersionConfigs{
 		DatagridComponent:   Datagrid73Component,
 		MySQLImageURL:       MySQL80ImageURL,
 		MySQLComponent:      MySQL80Component,
-		PostgreSQLImageURL:  PostgreSQL10ImageURL,
-		PostgreSQLComponent: PostgreSQL10Component,
+		PostgreSQLImageURL:  PostgreSQL12ImageURL,
```

Comment:
don't update the prior version.
And did you run make csv to update the csv?

Reply:
fixed